### ISSUE TITLE
NEW DESIGN: Update tables styles according to Figma mockup

### ISSUE THREAD

**Issue by stephanegigandet** [openfoodfacts/openfoodfacts-server#7411]

**Comment by sdeepanshu17** [comment]
Please assign this issue to me.

**Comment by stephanegigandet** [comment]
> Please assign this issue to me.

Hi @sdeepanshu17 , I just did.

**Comment by sdeepanshu17** [comment]
I will not be able to work on this issue. Please assign it to someone else. Sorry for the inconvenience caused.

**Comment by Edrees-creator** [comment]
is this issue still open?

**Comment by yesimhozman** [comment]
can you assign it to me?

**Comment by teolemon** [comment]
Yes it is @Edrees-creator , do you have a working dev environment yet ?

**Comment by G26karthik** [comment]
Hi! I'd like to work on this issue for Hacktoberfest 2025. 
I have experience with CSS/frontend work. Could you please assign it to me?
